### ISSUE TITLE
Accessible checkmarks

### ISSUE THREAD

**Issue by koendewilde** [Really-Simple-Plugins/complianz-gdpr#277]
Hello,

Some accessibility things came up today: 
The small revoke button and the preferences checkmarks are not accessible with keyboard navigation.

The accessibility testing tools are also giving warnings about the use of the labels with those checkmarks:
- There are labels inside labels. 
- There are labels with just a svg inside, they need some text.

_Thanks for the plugin (noticing more and more companies and semi-government in the Netherlands are changing to your plugin, must be doing something good :-)_

**Comment by rlankhorst** [comment]
Hi Koen, 

Thanks for your feedback, appreciate it! We have accessibility on the roadmap, will look at these specific issues shortly, although I think not all checkbox styles will be easily adjustable. Possibly we focus on the class checkbox style for accessibility purposes, as these are the most "basic".

**Comment by rlankhorst** [comment]
@koendewilde If you want you can create a pull request, pull requests are always welcome!

**Comment by rlankhorst** [comment]
Hi @koendewilde, we’ve update the plugin to make it wcag compatible. We’re going live Monday, but if you want you can check it out before that time, using github.

**Comment by koendewilde** [comment]
Thanks for the heads up @rlankhorst! And thanks for the WCAG update.

Just had an initial WCAG audit on a website and we almost passed :-) 

I think the main issue left concerning the Complianz plugin is that the revoke-button is not accessible by keyboard. (I am adding a custom revoke button to solve this for now).

**Comment by rlankhorst** [comment]
@koendewilde thanks, I'll add this to our to do list to look into.
